src/elf.rs

```diff
@@ -1,8 +1,16 @@
+#[cfg(feature = "zdebug")]
```

Comment:
Delete this and use `alloc::fmt` instead.

You'll also need to add `[macro_use]` to the `extern crate alloc` in lib.rs, and add `pub use std::fmt` in `mod alloc`.